scripts/CEE/hive-data-collection/script.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#
+# Collect data from Hive clusters in order to troubleshoot/debug cluster provisioning delays.
+# Input:
+#  (Env) CLUSTERID
+#        Description: The Internal Cluster ID for which data is being collected
+
+set -e
+set -o nounset
+set -o pipefail
+
+start_job(){
+    echo "Job started at $(date +"%Y-%m-%d %T")"
+    echo ".................................."
+    echo
+}
+
+finish_job(){
+    echo
+    echo ".................................."
+    echo "Job finished at $(date +"%Y-%m-%d %T")"
+}
+
+## Function which checks the clusterdeployment status
+## and print the details of each clusterdeployment
+clusterdeployment(){
+       
+    echo "### Checking clusterdeployment status..."
+    echo
+    oc get clusterdeployment -n uhc-production-${CLUSTER_ID}
```

Comment:
```suggestion
    oc get clusterdeployment -n "uhc-production-${CLUSTER_ID}"
```

The build is failing because this is not quoted so it could get expanded through globbing, etc. [See this](https://www.shellcheck.net/wiki/SC2086).

You'll need to change this in all of the places where you do this.